requirements/dev.txt

```diff
@@ -15,5 +15,5 @@ Sphinx==1.0.7
 # Testing
 nose==1.0.0
 mock==0.7.2
--e git://github.com/jbalogh/django-nose.git#egg=django_nose
--e git://github.com/jbalogh/test-utils.git#egg=test-utils
+git+git://github.com/jbalogh/django-nose.git#egg=django_nose
+git+git://github.com/jbalogh/test-utils.git#egg=test-utils
```

Comment:
I'm pretty sure the flag is still supported. It installs the packages in "develop" mode, and while I don't completely understand its effects, it's certainly not necessary.